sycl/test-e2e/Basic/windows_version_agnostic_sycl_lib.cpp

```diff
@@ -1,6 +1,6 @@
 // REQUIRES: windows
 
-// RUN: %clangxx --driver-mode=cl /std:c++17 /EHsc -I%sycl_include -I%opencl_include_dir %s -o %t.out /link /defaultlib:%sycl_static_libs_dir/sycl.lib
+// RUN: %clangxx -Wno-error=deprecated-declarations --driver-mode=cl /std:c++17 /EHsc -I%sycl_include -I%opencl_include_dir %s -o %t.out /link /defaultlib:%sycl_static_libs_dir/sycl.lib
```

Comment:
I think this is a dangerous change...

Reply:
According to agreement, I changed it to `-isystem`(in this case it is `/imsvc`  due to ` %clangxx --driver-mode=cl`)